config/initializers/prosopite.rb

```diff
@@ -0,0 +1,9 @@
+if Rails.env.local? && !IS_DOCKER
```

Comment:
The Prosopite docs suggest using `config/environments/development.rb` and `config/environments/test.rb` (as we were doing with Bullet), but I think that this more unified approach makes more sense.